pages/[slug]/login.tsx

```diff
@@ -0,0 +1,117 @@
+import { NextPage } from 'next'
+import Image from 'next/image'
+import Link from 'next/link'
+import logo from '../../public/devpleno.svg'

```

Comment:
NextJs has another way to use images instead import, this method converts automatically to correct sizes. 
Replace for this method please.